src/core/bittorrent/torrenthandle.cpp

```diff
@@ -1055,12 +1055,12 @@ int TorrentHandle::downloadPayloadRate() const
     return m_nativeStatus.download_payload_rate;
 }
 
-int TorrentHandle::totalPayloadUpload() const
+qlonglong TorrentHandle::totalPayloadUpload() const
```

Comment:
>Why did you use qlonglong here?

Do you mean "Why **didn't** you use"?
>could you comment on this, please?

Any comments? It's just my mistake. At the time I had processed a huge amount of code...
Why didn't I find this earlier? I use x64 environment where **int** and **qlonglong** are the same size.